release-process-documentation/release-team-guides/ci-signal-playbook.md

```diff
@@ -1,41 +1,141 @@
 
-- quirk: if a job is listed as FAILING but doesn't have "Overall" as one of its ongoing failures, it's not actually failing
-- if a job is failing in one of the meta-stages (Up, Down, DumpClusterLogs, etc), I find the owning sig
-- if a job is failing because a specific test case is failing, and that test case has a [sig-foo] in its name, I go bug sig-foo
-- with unit test case failures, I try to infer the sig from the path, or OWNERS files in it, otherwise I find the owning sig to help
-- with verify failures, I try to infer the failure from the log, otherwise I find the owning sig to help
-- if a test case is failing in one job consistently, but not others, both the job owner and test case owner are responsible for identifying why this combination is different
+### Feature Freeze to Code Slush
+
+Day to day tasks remain pretty much the same as before, with the following slight changes
+- Monitor [master-blocking](https://k8s-testgrid.appspot.com/sig-release-master-blocking) dashboard **daily**
+- Monitor [master-upgrade](https://k8s-testgrid.appspot.com/sig-release-master-upgrade) **every other day**
+- Starting now, curate a **weekly CI Signal report** escalating current test failures and flakes and send to kubernetes-dev@googlegroups.com community. [See below](#reporting-status) for more details on the report format and contents
+
+### Code Slush to Code Freeze
+
+This is when things really begin to ramp up in the release cycle with active bug triaging and followup on possible release blocking issues to assess release health. Day to day tasks of CI Signal lead include
+- Auditing test status of master-blocking, master-upgrade and release-1.x.y-blocking dashboards on **daily basis**
+- Keeping issues' status upto-date on GitHub
+- Working closely with SIGs, test owners, bug triage lead and Release team in triaging, punting and closing issues
+- Updating issue tracker frequently so anyone on the release team can get to speed and help followup on open issues
+- Continuing to send [weekly CI signal report](#reporting-status)
+
+### During Code Freeze
+
+- Continue best practices from Code slush stage. Monitor master-blocking, master-upgrade and release-1.x.y-blocking dashboards on **daily basis**
+- Quickly escalate any new failures to release team and SIGs
+- Continuing to send [weekly CI signal report](#reporting-status)
+
+### Exit Code Freeze
 
-I will ocasionally check [sig-release-master-upgrade](https://k8s-testgrid.appspot.com/sig-release-master-upgrade)
+- Once 1.x.y-rc.1 release branch is cut and master re-opens for next release PRs, stop monitoring master-blocking and **focus only on master-upgrade and 1.x.y-blocking dashboards on daily basis**
```

Comment:
IMO it's worth keeping some eye on master test CI, but this might more be a task for the release lead and branch manager in order to understand potential complexity and risk in cherry picking needed fixes from master back into the release branch during the final week(s).